spring-boot-starter-camunda/src/main/java/io/camunda/zeebe/spring/client/configuration/ConnectorConfiguration.java

```diff
@@ -5,34 +5,33 @@
 import io.camunda.connector.impl.config.ConnectorPropertyResolver;
 import io.camunda.connector.runtime.util.outbound.DefaultOutboundConnectorFactory;
 import io.camunda.connector.runtime.util.outbound.OutboundConnectorFactory;
-import io.camunda.zeebe.spring.client.connector.OutboundConnectorManager;
+import io.camunda.zeebe.spring.client.annotation.customizer.ZeebeWorkerValueCustomizer;
+import io.camunda.zeebe.spring.client.annotation.processor.AnnotationProcessorConfiguration;import io.camunda.zeebe.spring.client.connector.Filter.ZeebeConnectorFilter;import io.camunda.zeebe.spring.client.connector.OutboundConnectorManager;
```

Comment:
Please reformat the code to match the project code style, specifically to avoid multiple imports in one line